dependencies.props

```diff
@@ -44,7 +44,7 @@
     <!-- Scenario tests install this version of Microsoft.NetCore.App, then patch coreclr binaries via xcopy. At the moment it is
          updated manually whenever breaking changes require it to move forward, but it would be nice if we could update it automatically
          as we do with many of the package versions above -->
-    <BaselineMicrosoftNetCoreAppPackageVersion>2.1.0-preview2-26124-05</BaselineMicrosoftNetCoreAppPackageVersion>
+    <BaselineMicrosoftNetCoreAppPackageVersion>2.1.0-preview2-26131-06</BaselineMicrosoftNetCoreAppPackageVersion>
```

Comment:
How it this version number different from MicrosoftNETCorePlatformsPackageVersion ? Why can't we just use MicrosoftNETCorePlatformsPackageVersion ?

Reply:
Ok, I see. This is yet another way to mix and match bits. It is different from how we mix and match for corefx tests (https://github.com/dotnet/coreclr/pull/16263) so we will see different failures for it. I would nice to standardize on just one way to mix match so that we have just one configuration to worry about.

Reply:
Yeah, these scenario tests need a complete dotnet SDK to run. If this scheme was proving to be problematic my next step was likely going to assembling the shared\M.N.A\version directory using whatever scheme the test layout generator uses to build Core_Root. Its not an exact match for the real installer's shared\M.N.A\version directory but its probably close enough that the test wouldn't notice.